adafruit_platformdetect/chip.py

```diff
@@ -78,6 +79,9 @@ def _linux_id(self): # pylint: disable=too-many-branches
         if self.detector.check_dt_compatible_value("qcom,apq8016"):
             return APQ8016
 
+        if self.detector.check_dt_compatible_value("fu500"):
```

Comment:
We should probably change this to a little longer string like "sifive,fu540gsifive"

Reply:
Out of the 3 values in the proc file, only the first or third values work. 

if self.detector.check_dt_compatible_value("fu500"):                              # Works
if self.detector.check_dt_compatible_value("sifive,fu540gsifive"):           # Does not
if self.detector.check_dt_compatible_value("sifive"):                               # Works
if self.detector.check_dt_compatible_value("sifive,fu540gsifive,fu500"): # Does not

Reply:
What about:
```
if self.detector.check_dt_compatible_value("fu540gsifive"):
```